packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -401,6 +409,17 @@ class Tooltip extends OverlayClassMixin(ThemePropertyMixin(ElementMixin(PolymerE
         type: Object,
       },
 
+      /**
+       * Element used to link with the `aria-describedby`
+       * attribute. When not set, defaults to `target`.
+       * @protected
+       */
+      _ariaTarget: {
```

Comment:
`_effectiveAriaTarget`? I faintly remember we used the "effective" terminology in some places where there is a fallback.

Reply:
Thanks, renamed. There's also `_effectivePosition` in case of tooltip, which makes it aligned.